### ISSUE TITLE
linebreaks inside of provisioning files for dashboards

### ISSUE THREAD

**Issue by MikeColes** [sponsianus/dishy_grafana#10]
It appears line breaks have been introduced into the provisioning files for the Grafana dashboards.

Example of the line:
https://github.com/sponsianus/dishy_grafana/blob/d8068cddb328827be8fc1f83be5420f18a2f3000/grafana-provisioning/dashboards/network/netstats.json#L1519

Error shown in the logs after ```docker-compose up```
```
grafana_1   | logger=provisioning.dashboard type=file name=Starlink t=2022-06-17T00:29:57.349691511Z level=error msg="failed to load dashboard from " file=/etc/grafana/provisioning/dashboards/starlink/starlink.json error="invalid character '\\n' in string literal"
grafana_1   | logger=provisioning.dashboard type=file name=Network t=2022-06-17T00:29:57.349935725Z level=error msg="failed to load dashboard from " file=/etc/grafana/provisioning/dashboards/network/netstats.json error="invalid character '\\n' in string literal"
```

**Comment by sponsianus** [comment]
Thanks for the MR!! I just noticed it after rebuilding the templates. I don't check this account as much as I maybe should.
A user on Reddit was having trouble with a new install and I'm pretty sure I broke the templates on the last update. I cleaned up all the errors on Grafana startup and updated the template files. I'm not sure if the changes in your MR are still valid. Installs should be working as expected now.
